pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/TransactionsBase.java

```diff
@@ -432,4 +433,27 @@ protected void validateTopicName(String property, String namespace, String encod
             throw new RestException(Response.Status.PRECONDITION_FAILED, "Topic name is not valid");
         }
     }
+
+    protected void internalScaleTransactionCoordinators(AsyncResponse asyncResponse, int replicas) {
```

Comment:
We should define this method as `protected CompletableFuture<Void> internalScaleTransactionCoordinators`,
which means we can't pass `AsyncResponse` into the method.

Reply:
Yes, agree. It's a great practice.